src/Faker/Extension/ContainerBuilder.php

```diff
@@ -62,6 +62,7 @@ public static function defaultExtensions(): array
     {
         return [
             FileExtension::class => Core\File::class,
+            MedicalExtension::class => Core\Medical::class,
```

Comment:
When you rename this to `BloodExtension`, then make sure this is in alphabetical order. It would please my OCD =)

Reply:
How about `Medical\BloodExtension`?

Reply:
Looking at the [list of the current features](https://github.com/Nyholm/next-faker/blob/master/PROVIDERS.md) we have, there is nothing else that would fit in `Medical` namespace.

Reply:
We could add a list of deceases, but that would be localized.

Reply:
I would rather keep the list of features as small as possible. 

I am more than happy to add all kinds of medical related stuff in a separate package. Ie `fakerphp/medical`. 

Lets focus on the current features and this PR at the moment and later we can discuss if this belongs to core or not

Reply:
I would like to keep this in a fakerphp/medical, this doesn’t feel like something that every application would need. Which is kind of the point of our “core”, to keep general and commonly needed extensions.

Reply:
I doubt that last one a bit since it brings the ease of use. Nobody wants to have 6 extra dependencies for a package like this. 

I ofc agree to split of all the language specifics where it is impossible to maintain centralised and also all the language packs holds a lot of storage but for all other generic stuff?

Since for example when i need to create a dummy person with all its info i need a for example Swedish pack, Faker core and a medical extension to add its blood type